extensions/guacamole-auth-json/src/main/resources/translations/fr.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "DATA_SOURCE_JSON" : {
+        "NAME" : "JSON"
```

Comment:
Is this the correct French translation for this? Or just a copy of the English translation?